packages/api/src/routers/svc/emails.ts

```diff
@@ -36,13 +36,7 @@ interface EmailMessage {
 }
 
 function isEmailMessage(data: any): data is EmailMessage {
-  return (
-    'from' in data &&
-    'to' in data &&
-    'subject' in data &&
-    'html' in data &&
-    'text' in data
-  )
+  return 'from' in data && 'to' in data && 'subject' in data
```

Comment:
I wonder if subject is even required :-/

Reply:
It should be optional too. Let me change that